messagebus/src/main/java/com/yahoo/messagebus/Sequencer.java

```diff
@@ -147,10 +162,16 @@ public void handleReply(Reply reply) {
             }
         }
         if (msg != null) {
-            sequencedSend(msg);
+            Boolean alreadySending = isSending.get();
+            if ((alreadySending == Boolean.TRUE) && (msn != null)) {
```

Comment:
Could probably use auto-(un)boxing  of bools for these if desired, i.e. `if (alreadySending && ...)`, `isSending.set(true)`. etc

Reply:
It will be null first time, unless .withInitial

Reply:
Done